tests_e2e/playwright/conftest.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import os
+
+import dotenv
+import pytest
+from navigator import Navigator
+
+logger = logging.getLogger()
+
+
+@pytest.fixture(scope="session")
+def _navigator_session(browser_name):
+    """Set up a navigator instance, login with username/password, start
+    a server. Teardown when session is complete.
+    Do not use this for individual tests, use `navigator` fixture
+    for tests."""
+    dotenv.load_dotenv()
+    # try/except added here in attempt to reach teardown after error in
+    # order to close the browser context which will save the video so I debug
+    # the error.
+    try:
+        nav = Navigator(
+            nebari_url=os.environ["NEBARI_FULL_URL"],
+            username=os.environ["USERNAME"],
+            password=os.environ["PASSWORD"],
+            headless=False,
+            browser=browser_name,
+            auth="password",
+            instance_name=os.environ["INSTANCE_NAME"],
+            video_dir="videos/",
+        )
+        nav.login_password()
+        nav.start_server()
+        yield nav
+    except Exception as e:
```

Comment:
I was hit with a timeout error here quite often while stepping through this with a debugger. We need an option to increase the overall timeout or disable it completely to enable local development.

Reply:
With `pytest-playwright` installed, I see this in the help message:

```
Interrupt test run and dump stacks of all threads after a test times out:
  --timeout=TIMEOUT     Timeout in seconds before dumping the stacks.  Default is 0 which
                        means no timeout.
  --timeout_method={signal,thread}
                        Deprecated, use --timeout-method
  --timeout-method={signal,thread}
                        Timeout mechanism to use.  'signal' uses SIGALRM, 'thread' uses a timer
                        thread.  If unspecified 'signal' is used on platforms which support
                        SIGALRM, otherwise 'thread' is used.
```

Could we maybe use that?